mscore/pagesettings.cpp

```diff
@@ -53,11 +53,14 @@ PageSettings::PageSettings(QWidget* parent)
       for (int i = 0; i < QPageSize::LastPageSize; ++i)
             pageGroup->addItem(QPageSize::name(QPageSize::PageSizeId(i)), i);
 
+      buttonBox->addButton(buttonApplyToAllParts, QDialogButtonBox::ApplyRole);
```

Comment:
Please add `buttonApplyToAllParts->setEnabled(!cs->isMaster());` here

Reply:
The line buttonApplyToAllParts->setEnabled(!cs->isMaster()); is already present. It appears in void PageSettings::setScore(Score* s) rather than in the Constructor as it does in editstyle.cpp